modules/app_gke/variables.tf

```diff
@@ -65,6 +65,11 @@ variable "enable_private_gke_nodes" {
   description = "Enable private nodes for the GKE cluster."
 }
 
+variable "enable_gcs_fuse_csi_driver" {
```

Comment:
should we default this to false

Reply:
Omg definitely. Sorry, i meant to disable by default

Reply:
Fixed